docs/docs/guides/api-usage.md

```diff
@@ -0,0 +1,307 @@
+---
```

Comment:
1. Some of the paths in this docs page are broken, e.g. still uses the old `/v1/organizations` prefix
2. Can you raise this docs page as a separate PR? I would like to merge the refactors here ASAP since otherwise we'll get lots of merge conflicts with a diff this big.